src/applications/disability-benefits/view-payments/utils.js

```diff
@@ -1,5 +1,53 @@
+import React from 'react';
+import Telephone, {
+  CONTACTS,
+  PATTERNS,
+} from '@department-of-veterans-affairs/formation-react/Telephone';
+
 const SERVER_ERROR_REGEX = /^5\d{2}$/;
 const CLIENT_ERROR_REGEX = /^4\d{2}$/;
 
 export const isServerError = errCode => SERVER_ERROR_REGEX.test(errCode);
 export const isClientError = errCode => CLIENT_ERROR_REGEX.test(errCode);
+
+export const ClientErrorAlertContent = (
+  <>
+    <h2 className="vads-u-margin-y--0 vads-u-font-size--lg">
+      We don’t have a record of VA payments made to you
+    </h2>
+    <p>
+      We can't find any VA payments made to you. Some details may not be
+      available online. For example, payments less than $1 for direct deposit,
+      or $5 for mailed checks, will not show in your online payment history.
+    </p>
+    <p>
+      VA pays benefits on the first day of the month for the previous month.
+      Please wait at least 3 business days (Monday-Friday) before reporting
+      non-receipt of a payment.
+    </p>
+  </>
+);
+
+export const ServerErrorAlertContent = (
+  <>
+    <h2 className="vads-u-margin-y--0 vads-u-font-size--lg">
```

Comment:
I think it makes more sense to use `vads-u-font-size--h3` here, then we know the expected size.